app/templates/about.jade

```diff
@@ -83,7 +83,7 @@ block content
             .team-bio
               h6.label.team-name
                 a(href="http://bryukh.com" rel="external") Valentin Bryukhanov
-              small Senior Gameplay Developer
+              small(data-i19n="about.bryukh_title")
```

Comment:
I think there's a small typo: 
```suggestion
              small(data-i18n="about.bryukh_title")
```

Reply:
Hahaha wow! Thank you for spotting this :)